src/stylus/fablevision/fv-gene-label.styl

```diff
@@ -71,6 +71,21 @@
     justify-content: flex-start
     padding-right: 30px
 
+  &.fv-layout-a .geniblocks.chromosome-container .items.rtl .labels .fv-gene-label
```

Comment:
This is pretty ugly - I really had to force some labels into the correct places in different ways on different layouts